test/operations/manager.go

```diff
@@ -168,11 +169,35 @@ func (m *Manager) SetGenesis(genesisActions []*state.GenesisAction) error {
 
 	_, err = m.st.SetGenesis(m.ctx, genesisBlock, genesis, dbTx)
 
-	err = dbTx.Commit(m.ctx)
+	errCommit := dbTx.Commit(m.ctx)
+	if errCommit != nil {
+		return errCommit
+	}
+
+	return err
+}
+
+func (m *Manager) SetForkID(forkID uint64) error {
```

Comment:
I understand this function is for testing propouses, isn't it? if it's the case please add a comment line just before the function signature saying this

Reply:
Yes, all functions inside test folder are for testing purposes